common-gradle-enterprise-bazel-configuration/.bazelrc

```diff
@@ -0,0 +1,21 @@
+# To publish Build Scans to your Develocity server, add the following to the .bazelrc file in your 
+# build workspace:
+common:develocity --remote_cache=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --bes_results_url=https://<<develocity-server>>/build/ # adjust to your Develocity server
+common:develocity --bes_backend=grpcs://<<develocity-server>> # adjust to your Develocity server
+common:develocity --build_event_publish_all_actions=true
+common:develocity --noslim_profile
+
+build --config=develocity
+query --config=develocity
+
+
+# Because Bazel build scans are enriched by allowing users to upload data to the remote cache, the following
+# additional steps are required:
+# 1. Log into your Develocity server, from the upper-right corner click your user icon, then "My Settings".
+# 2. Ensure that your user has at least "Publish build scans" and either "Read and write build cache data" or 
+#    "Read build cache data and write Bazel CAS data" permissions.
+# 3. Select "Access keys" on the left and then generate a new access key.
+# 4. In a .bazelrc file in your user’s home directory, add the following:
+#    common:develocity --remote_cache_header=Authorization=<<access key generated>>
+#    common:develocity --bes_header=Authorization=<<access key generated>>
```

Comment:
Make sure to watch this ticket for when we fix it: https://github.com/gradle/ge/issues/29905

Reply:
On second thought, let's remove this entire section. We don't describe configuring the access keys for either Gradle or Maven and it also isnt part of the Gradle or Maven samples.

Reply:
removed